# Arena locations
# - Collector bin   = [0.00 0.25 0.48 1.90]
# - Starting Zone B = [0.48 0.00 2.13 1.90] Center = [1.305 0.95]
# - Starting Zone A = [2.13 0.00 3.78 1.90] Center = [2.955 0.95]
# - Obstacle Zone   = [0.00 1.90 3.78 4.64]
# - Mining Zone     = [0.00 4.64 3.78 7.38]

timers:
  finish_dig_1:
    m: 4.0
    s: 0.0
  finish_dig_2:
    m: 8.0
    s: 0.0

paths:
  dig_1:
    - x: 2.0 #1.5
      y: 1.4
      theta: 0.0
      reverse: false
      match_orientation: false
      drive_profile: 0
    -
      x: 2.0 #2.0 #1.5
      y: 5.75
      theta: 0.0
      reverse: false
      match_orientation: false
      drive_profile: 0

  hopper_1:
    -
      x: 2.5
      y: 1.4
      theta: 0.0
      reverse: true
      match_orientation: false
      drive_profile: 0
    -
      x: 1.04 # 1.00
      y: 1.4
      theta: 0.0
      reverse: true
      match_orientation: false
      drive_profile: 2

  dig_2:
    - x: 2.0 #1.5
      y: 1.4
      theta: 0.0
      reverse: false
      match_orientation: false
      drive_profile: 3
    -
      x: 2.0 #2.0 #1.5
      y: 5.5
      theta: 0.0
      reverse: false
      match_orientation: false
      drive_profile: 0

  hopper_2:
    -
      x: 2.5
      y: 1.4
      theta: 0.0
      reverse: true
      match_orientation: false
      drive_profile: 0
    -
      x: 1.04
      y: 1.4
      theta: 0.0
      reverse: true
      match_orientation: false
      drive_profile: 2

  final_position:
    -
      x: 2.0
      y: 1.4
      theta: 0.0
      reverse: false
      match_orientation: false
      drive_profile: 3